### PR TITLE
add support to Ipsec_pfs field be set with "disabled" and add example files

### PR DESCRIPTION
**Pull request by zhengjc2018**

### CODE REVIEW

File: alicloud/resource_alicloud_vpn_connection.go [hunk 1]
```diff
@@ -153,7 +153,7 @@ func resourceAliyunVpnConnection() *schema.Resource {
 							Type:         schema.TypeString,
 							Optional:     true,
 							Default:      VPN_PFS_G2,
-							ValidateFunc: validateAllowedStringValue([]string{VPN_PFS_G1, VPN_PFS_G2, VPN_PFS_G5, VPN_PFS_G14, VPN_PFS_G24}),
+							ValidateFunc: validateAllowedStringValue([]string{VPN_PFS_G1, VPN_PFS_G2, VPN_PFS_G5, VPN_PFS_G14, VPN_PFS_G24, VPN_PFS_DISABLED}),
```
Comment (xiaozhu36) [inline alicloud/resource_alicloud_vpn_connection.go]: Please add the new value in the docs.

File: alicloud/resource_alicloud_vpn_connection.go [hunk 2]
```diff
@@ -134,7 +134,7 @@ func resourceAliyunVpnConnection() *schema.Resource {
 			"ipsec_config": {
 				Type:     schema.TypeList,
 				Optional: true,
-				Computed: true,
+				// Computed: true,
```
Comment (xiaozhu36) [inline alicloud/resource_alicloud_vpn_connection.go]: why remove the line 137?

File: examples/vpn-connection/main.tf [hunk 3]
```diff
@@ -0,0 +1,38 @@
+resource "alicloud_vpc" "default" {
```
Comment (xiaozhu36) [inline examples/vpn-connection/main.tf]: please run `terraform fmt` before committing the code.

**Comment by zhengjc2018** [comment]
i think that "computed:true" means the value can not be set, only can know
after apply. But actually, it can be set as normal arguments and has its
default value.So i remove the line 137

He Guimin <notifications@github.com> 于2019年8月22日周四 下午8:40写道：

**Comment by zhengjc2018** [comment]
fix fmt error and fix docs error

**Pull request merged**